addons/im_livechat/controllers/main.py

```diff
@@ -76,9 +76,9 @@ def _get_guest_name(self):
 
     @http.route('/im_livechat/get_session', methods=["POST"], type="jsonrpc", auth='public')
     @add_guest_to_context
-    def get_session(self, channel_id, previous_operator_id=None, chatbot_script_id=None, persisted=True):
-        agent = request.env["res.users"]
-        chatbot_script = request.env["chatbot.script"]
+    def get_session(self, channel_id, operator_lookup_params=None, persisted=True):
```

Comment:
Some calls have been updated to use the channel_info object (as has the CORS override), but the original method hasn't been?